main/bash/bash.pre-deinstall

```diff
@@ -1,3 +1,3 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
please revert this back to `#!/bin/sh`.

Reply:
I did so, to remove package dependency to /bin/sh. But it really have no much sense, so I revert it.